ocs_ci/framework/__init__.py

```diff
@@ -455,6 +456,21 @@ def switch_to_cluster_by_cluster_type(self, cluster_type, num_of_cluster=0):
             self.get_cluster_type_indices_list(cluster_type)[num_of_cluster]
         )
 
+    def run_for_all_clusters(self, func):
```

Comment:
please check if `get_client_contexts_if_available` suits for this job and can be reused in the body of this wrapper